app/helpers/application_helper.rb

```diff
@@ -521,6 +521,19 @@ def notifications
     end
   end
 
+  def flash_notifiations
```

Comment:
Where is this used?

Reply:
It was supposed to be a helper to only get non-inline flashes. Removed it now.